cameo/strain_design/deterministic/linear_programming.py

```diff
@@ -269,9 +269,7 @@ def run(self, max_knockouts=5, biomass=None, target=None, max_results=1, *args,
 
                 if len(knockouts) < max_knockouts:
                     self._number_of_knockouts_constraint.lb = self._number_of_knockouts_constraint.ub - len(knockouts)
-
-                tm(do=partial(self._model.solver.add, integer_cut),
-                   undo=partial(self._model.solver.remove, integer_cut))
+                self._model.add_cons_vars(integer_cut)
```

Comment:
shouldn't we use `add` and `remove` only and let the solver handle the adding of constraints and variables.